dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java

```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
 	
 	
 	
-	public String inodeCount = "select count(inode) as inode_count from field where inode = ?";
+	public final String inodeCount = "select count(inode) as inode_count from field where inode = ?";
 	
 	
 	
-	public String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
+	public final String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
 	
 	
-	public String selectCountOfType= "select count(*) as test from field where structure_inode = ? and field_type like ? or field_type like ?";
+	public final String selectCountOfType= "select count(*) as test from field where structure_inode = ? and field_type like ? or field_type like ?";
 	
-	public String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
-	public String selectFieldVar= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where id = ? ";
-	public String selectFieldVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where variable_key = ? ";
-	public String selectFieldIdVarByKey= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? and variable_key = ?";
+	public final String selectFieldVars= 	"select id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date from field_variable where field_id = ? order by variable_key";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)